src/temp/CarbonCapture/CarbonCapture.tsx

```diff
@@ -0,0 +1,192 @@
+  byProject: string;
+  sitePotential: string;
+}
+
+type carbonCapture = Omit<TabPanelProps, 'index' | 'value'>;
+
+function CustomTabPanel(props: TabPanelProps) {
+  const {
+    value,
+    index,
+    beforeIntervation,
+    byProject,
+    sitePotential,
+    ...other
+  } = props;
+  const { t } = useTranslation(['projectDetails']);
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      className={style.carbonCaptureMainContainer}
+      {...other}
+    >
+      {value === index && (
+        <>
+          <div className={style.carbonCaptureInfoContainer}>
+            <div className={style.carbonCaptureLabelMainContainer}>
+              <div className={style.carbonCaptureLabelContainer}>
+                <div className={style.carbonCapture}>
+                  {t('projectDetails:totalCO₂Captured')}
+                </div>
+                <div className={style.unit}>(tons)</div>
+              </div>
+              <div className={style.infoIconContainer}>
+                <InfoIcon />
+              </div>
+            </div>
+            <div className={style.carbonCatureDetailContainer}>
+              <div className={style.carbonCaptureIndicator}>
+                <div className={style.beforeIntervationIndicator} />
+                <div className={style.byProjectIndicator} />
+                <div className={style.projectPotential} />
+              </div>
+              <div className={style.carbonCaptureDataContainer}>
+                <div>
+                  <p className={style.beforeIntervationData}>
+                    {beforeIntervation}
+                  </p>
+                  <p className={style.beforeIntervationLabel}>
+                    {t('projectDetails:beforeIntervention')}
+                  </p>
+                  <p className={style.beforeIntervationDate}>
+                    {t('projectDetails:before', {
+                      date: 2018,
+                    })}
+                  </p>
+                </div>
+                <div className={style.byProjectMainConatiner}>
+                  <p className={style.byProjectData}>{byProject}</p>
+                  <p className={style.byProjectLabel}>
+                    {t('projectDetails:byProject')}
+                  </p>
+                  <p className={style.byProjectDate}>
+                    {t('projectDetails:since', {
+                      date: 2018,
+                    })}
+                  </p>
+                </div>
+                <div className={style.sitePotentialDataContainer}>
+                  <p className={style.sitePotentialData}>{sitePotential}</p>
+                  <p className={style.sitePotentialLabel}>
+                    {t('projectDetails:sitePotential')}
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+          <button className={style.seeMoreButton}>
```

Comment:
Add the functionality of see more button.
Also, add see less button as well.
Refer this PR: https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1973